diff --git a/include/cereal/details/static_object.hpp b/include/cereal/details/static_object.hpp
--- a/include/cereal/details/static_object.hpp
+++ b/include/cereal/details/static_object.hpp
@@ -94,6 +94,7 @@ namespace cereal
             std::unique_lock<std::mutex> lock;
           #else
           public:
+            LockGuard() = default;
	          LockGuard(LockGuard const &) = default; // prevents implicit copy ctor warning
             ~LockGuard() CEREAL_NOEXCEPT {} // prevents variable not used
           #endif
